candid_relate/src/lib.rs

```diff
@@ -0,0 +1,103 @@
+extern crate candid;
+
+use std::rc::Rc;
+
+pub type Type = candid::parser::types::IDLType;
+pub type Label = candid::parser::types::Label;
```

Comment:
You may be using an outdated code. `Label` has been moved to `candid::types::Label`. In general, we prefer to use `candid::types::Type` when referring to candid types. `candid::parser::types::*` is only an AST, and come be converted to `types::Type` via `candid::parser::typing::check_prog`. The benefit of this design is that we get a consistent view for both types derived from Rust and from candid file.